src/adapters/MeetingsJSONAdapter.js

```diff
@@ -201,6 +227,88 @@ export default class MeetingsJSONAdapter extends MeetingsAdapter {
     return stream;
   }
 
+  /**
+   * Toggles muting the local audio media stream track.
+   * Used by "mute-audio" meeting control.
+   *
+   * @param {string} ID  ID of the meeting for which to mute local audio.
+   * @memberof MeetingsJSONAdapter
+   * @private
+   */
+  async toggleMuteAudio(ID) {
+    if (this.datasource[ID]) {
+      const meeting = this.datasource[ID];
```

Comment:
I know we fixed it in the hook, but I think it's good to show "proper" code
```suggestion
      const meeting = {...this.datasource[ID]};
```